src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -145,9 +145,9 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
       'mfa-token-serial': [''],
       'mfa-token-pin': [''],
       lock_enabled: [{ value: false, disabled: this.editing }],
-      encryption_enabled: [null],
+      encryption_enabled: [{ value: null, disabled: !this.kmsConfigured && !this.s3Configured }], 
       encryption_type: [
-        null,
+        { value: null, disabled: !this.kmsConfigured && !this.s3Configured }, 
```

Comment:
This is a radio button so if s3 in not configured, we need to make sure s3 radio is disabled.
Similarly, when kms is not configured it should be disabled.
This is different from `encryption_enabled` which is a higher level check ensuring either one of them is present.
And I think its incorrect right now. So need to check. (see above comment)